protected/humhub/modules/user/models/forms/AccountChangeUsername.php

```diff
@@ -89,4 +90,15 @@ public function sendChangeUsername()
         return true;
     }
 
+    /**
+     * Validate attribute newUsername
+     * @param string $attribute
+     */
+    public function validateForbiddenUsername($attribute, $params)
+    {
+        if(in_array($this->$attribute, Yii::$app->controller->module->forbiddenUsernames)){
```

Comment:
Would be nice if we could somehow check this without case sensitive characters.

e.g. in the blacklist everything in lower case and here the attribute strtolower